rules.txt

```diff
@@ -2758,25 +2759,32 @@
 /sphs                                     301  /programs/community-schools/south-philadelphia-high-school/
 /spotlight                                301  /the-latest
 /spotlight/love-park                      301  /spotlight/summer-of-love/
+/spotlight/career-empowerment-fair/       301  /departments/office-of-the-chief-administrative-officer/office-of-talent-and-employee-success/
 /spotlight/city-of-philadelphia-holiday-celebrations/ 301  /spotlight/holiday-celebrations
 /spotlight/city-of-philadelphia-holiday-celebrations-2019/ 301  /spotlight/holiday-celebrations
+/spotlight/city-of-philadelphia-holiday-celebrations-2020/  301  /holiday-celebrations
 /spotlight/city-of-philadelphia-holiday-celebrations-2021/ 301  /spotlight/holiday-celebrations
 /spotlight/city-of-philadelphia-holiday-celebrations-2022/ 301  /spotlight/holiday-celebrations
 /spotlight/city-of-philadelphia-holiday-celebrations-2023/ 301  /spotlight/holiday-celebrations
 /spotlight/christmas-village-in-love-park-2019/	301  /departments/philadelphia-parks-recreation/
 /spotlight/fall-in-love-2019/	            301  /departments/philadelphia-parks-recreation/
 /spotlight/fall-in-love-2021/	            301  /departments/philadelphia-parks-recreation/
+/spotlight/immigrant-business-week/?.*    301  /departments/department-of-commerce/
 /spotlight/immigrant-heritage-month       301  /immigrant-heritage-month
 /spotlight/immigrant-heritage-month-2021  301  /immigrant-heritage-month
 /spotlight/immigrant-heritage-month-2022  301  /immigrant-heritage-month
 /spotlight/immigrant-heritage-month-2023  301  /immigrant-heritage-month
+/spotlight/juneteenth-2021/               301  /phila.gov
```

Comment:
Here the path that you are redirecting to isn't /phila.gov, but it's actually just the slash / since that is the path after the domain. 

So this line should read
/spotlight/juneteenth-2021/ 301 /

As written, this would redirect people to phila.gov/phila.gov.